dodo.py

```diff
@@ -13,3 +15,73 @@ def task_ecosystem_setup():
         "conda update conda",
         "conda install anaconda-project 'tornado<5.0'",
     ]}
+
+
+name_param = {
+    'name': 'name',
+    'long': 'name',
+    'type': str,
+    'default': 'attractors'
+}
+
+def _prepare_paths(root, name, test_data):
+    if root == '':
+        root = os.getcwd()
+    root = os.path.abspath(root)
+    test_data = test_data if os.path.isabs(test_data) else os.path.join(root, test_data)
+    return {
+        'real': os.path.join(root, name, 'data'),
+        'test': os.path.join(test_data, name),
+    }
+
+
+def task_small_data_setup():
+    """Copy small versions of the data from test_data"""
+
+    def copy_test_data(root='', name='attractors', test_data='test_data'):
+        print('Copying test data for {} ...'.format(name))
+        paths = _prepare_paths(root, name, test_data)
+
+        if not os.path.exists(paths['test']):
+            print("Fail: No test_data found for {} in {}".format(name, paths['test']))
```

Comment:
How will these failing conditions be detected by the tests, if they are just printing "Fail"?

Reply:
They won't. This was for debugging purposes. I should be raising proper errors.